app/models/room.rb

```diff
@@ -53,4 +57,8 @@ def create_meeting_options
       RoomMeetingOption.create(room: self, meeting_option: option, value: option.default_value)
     end
   end
+
+  def find_meeting_option_by_name(name)
+    room_meeting_options.joins(:meeting_option).find_by('meeting_option.name': name)
+  end
```

Comment:
I would move this method to `MeetingOption`

Reply:
Wouldn't that make `  anyone_joins_as_moderator?`:
```
  def anyone_joins_as_moderator?
    MeetingOption.find_room_meeting_optionby_name(name: 'glAnyoneJoinAsModerator', room_id: id)&.value == 'true'
  end
```
Is that preferred?